src/Appwrite/Migration/Version/V19.php

```diff
@@ -640,6 +646,21 @@ private function getFunctionCommands(Document $function): string
         return $commands;
     }
 
+    private function migrateEnumAttributeSize(): void
+    {
+        \Co\run(function () {
+            foreach ($this->documentsIterator('attributes') as $attribute) {
+                go(function ($attribute) {
+                    $attribute->setAttribute('size', Database::LENGTH_KEY);
```

Comment:
We should check the current size and make sure we're not making it smaller than what it was before.

Reply:
Added condition for checking current size, although something else will be needed to solve the bug. I think we should truncate the value and update the value to maximum 255 length size. 
cc @abnegate

Reply:
Actually I think it should be okay to ignore ones that are longer than 255 already 🤔 any developers in that scenario just won't be able to add any _more_ that are longer than 255

Reply:
They will be able to update those enum attributes with length greater than 255. Update endpoint doesn't restrict size.

Reply:
Okay. resolving this then